package.json

```diff
@@ -32,12 +32,12 @@
     "server": "pubsweet server",
     "setupdb": "pubsweet setupdb",
     "start": "docker-compose up",
-    "start:services": "docker-compose up postgres fakes3",
+    "start:services": "docker-compose up -d postgres fakes3 api-dummy",
     "start:styleguide": "docker-compose run --no-deps -p 6060:6060 app yarn run styleguide",
     "stop:services": "docker-compose down -v",
     "styleguide": "styleguidist server",
     "test": "NODE_CONFIG_ENV=unit-test jest --runInBand",
-    "test:browser": "NODE_CONFIG_ENV=test NODE_ENV=production DEBUG=replay testcafe --concurrency 1 \"${BROWSER:-chrome}\" 'test/**/*.browser.js'",
+    "test:browser": "NODE_CONFIG_ENV=test NODE_ENV=production testcafe --speed 1 --concurrency 1 \"${BROWSER:-chrome}\" 'test/**/*.browser.js'",
```

Comment:
does `--speed 1` mean slower or faster?

Reply:
speed 1 is the fastest testcafe will run but is also the default so can probably be removed

Reply:
See #1931